.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Issue Template
+about: Create a report to help us fix bugs/inaccuracies
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**TL;DR**
+Check if your issue answers those question:
+
+- Does my issue clearly describe what exactly is not working properly?
+- Does my issue clearly describe what are my expectations and what is the current output?
+- Did I attach logs/screenshots/errors which ocured?
+- Did I provide information how I run the code/library/application and on which platform/browser?
```

Comment:
This could actually be a checklist, but I'm just nitpicking 😄

Reply:
Done